### DISCUSSION TITLE
Pull request comments can be edited by other users!

### DISCUSSION THREAD

**Discussion by MrXavier** [community/community#41659] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

In Github.com pull requests, a comment can be edited by users that didn't created the comment. Then I can edit a comment that another user created.
This issue is critical because it hurts the consistency of comments and its owners.

See image bellow as example. This is actually a comment my colleague created. I can click on edit button and change its content, even not being the owner of that comment.
![image](https://user-images.githubusercontent.com/7959331/207424182-5a9e64c0-4fb0-4ceb-8aa2-ca1c69180bcd.png)

Feel free to try it out to reproduce it. Let me know if you have any questions.

**Comment by xCENTx** [comment]
Somebody I was working on a private repository with ended up posting the source public. I made a PR letting people know there were multiple people involved with the making of said source (He claimed to have used it to get a job in an interview)

Stuff like this is so abusive and i have NO IDEA why github thought this would be a good idea
![image](https://user-images.githubusercontent.com/80198020/210949874-8cf6b02e-df9d-4d76-87c6-b2e1a164cc90.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MrXavier ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
